### PR TITLE
chore: move @internal tokens into component token

### PR DESCRIPTION
**Pull request by afc163**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [x] Refactoring
- [x] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link

https://github.com/ant-design/ant-design/pull/47909#discussion_r1527135972

### 💡 Background and solution

下面的 XxxxToken 就是用来放暂时不对外暴露的 token 的，没必要使用 @internal 在 ComponentToken 里。

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |      --     |
| 🇨🇳 Chinese |   --        |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/modal/style/index.ts [hunk 1]
```diff
@@ -72,19 +84,6 @@ export interface ComponentToken {
   confirmBtnsMarginTop: string | number;
```
Comment (afc163) [inline components/modal/style/index.ts:84]: 图片

这段逻辑很奇怪，为了 wireframe 的逻辑在这里加了一些额外的 token 定义。

Comment (afc163) [reply components/modal/style/index.ts:84]: 不如单独搞一个 wireframe 的算法。

**Comment by MadCcc** [comment]
boolean 没有办法给 css var 运算，只能全部放进 component token 里，用 js 逻辑算出结果

**Comment by afc163** [comment]
@MadCcc 现在这个逻辑太脏了，还污染了 ComponentToken。

**Pull request closed without merging**